src/main/java/org/cbioportal/web/columnar/BasicDataBinner.java

```diff
@@ -85,9 +99,22 @@ public <T extends DataBinCountFilter, S extends DataBinFilter, U extends DataBin
         switch (dataBinCountFilter) {
             // TODO: first case is to support clinical data, but clinical data is not using this now. We should update controller to use this method later
             case ClinicalDataBinCountFilter clinicalDataBinCountFilter -> {
```

Comment:
curious if this would work to split up customdata
 case ClinicalDataBinCountFilter clinicalDataBinCountFilter when customDataService.getCustomDataSessions(uniqueKeys).isEmpty()